mat/format.go

```diff
@@ -243,20 +243,20 @@ func formatMATLAB(m formattable, prefix string, _ int, _ byte, squeeze bool, fs
 			fmt.Fprintf(fs, "%%!%c(%T=Dims(%d, %d))", c, m.Mat(), rows, cols)
 			return
 		}
-		fs.Write([]byte{'['})
+		fmt.Fprint(fs, "[")
```

Comment:
I'd prefer not to make these changes; they are more heavy weight. `fmt` depends on `reflect` and interpretation of types while write just spits bytes which the compiler has nicely prepared.

Reply:
This was helpful, thanks.